scripts/build-meetings.js

```diff
@@ -40,14 +40,14 @@ async function buildMeetings() {
     });
 
     const eventsForHuman = JSON.stringify(eventsItems, null, '  ');
-    // console.log('The following events got fetched', eventsForHuman);
-
-    writeFileSync(
-      resolve(__dirname, '../config', 'meetings.json'),
-      eventsForHuman
-    );
-  } catch (e) {
-    console.error(e);
+    console.log('The following events got fetched', eventsForHuman);
+
+    writeFileSync(writePath,eventsForHuman);
+  } catch (err) {
+    throw new Error(err)
   }
 }
-buildMeetings();
+
+buildMeetings(resolve(__dirname, '../config', 'meetings.json')).catch(err => console.error(err));
```

Comment:
why are we catching errors while calling the function? Aren't the errors being handled inside the function?